mimic/rest/maas_api.py

```diff
@@ -30,6 +30,16 @@
 Request.defaultContentType = 'application/json'
 
 
+def _find_index_of_id(xs, val):
```

Comment:
Noted. I'm used to JSDoc where `list#index` is a method and `list.index` is a static class method, in case you're wondering where that's from.